lib/rubocop/cop/util.rb

```diff
@@ -271,6 +271,15 @@ def compatible_external_encoding_for?(src)
         src = src.dup if RUBY_VERSION < '2.3'
         src.force_encoding(Encoding.default_external).valid_encoding?
       end
+
+      def to_supported_styles(enforced_style)
```

Comment:
I don't like this method that much - too many things seem hardcoded in it, which means people are likely going to forget updating it and me might end up with more unvalidated config down the road.